Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9
+ADD app.py .
+ADD requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

[notice] A new release of pip is available: 23.0.1 -> 24.0
[notice] To update, run: pip install --upgrade pip

Essaye de voir pour ajouter un virtual env pour pip